core/src/test/java/org/elasticsearch/search/aggregations/AggregatorParsingTests.java

```diff
@@ -112,28 +112,29 @@ public static void init() throws IOException {
             (b) -> {
                 b.bind(Environment.class).toInstance(new Environment(settings));
                 b.bind(ThreadPool.class).toInstance(threadPool);
-            }, settingsModule
-            , scriptModule, new IndicesModule(namedWriteableRegistry) {
+                b.bind(ScriptService.class).toInstance(scriptModule.getScriptService());
+            },
+            settingsModule,
+            new IndicesModule(namedWriteableRegistry) {
+                @Override
+                protected void configure() {
+                    bindMapperExtension();
+                }
+            }, new SearchModule(settings, namedWriteableRegistry) {
+                @Override
+                protected void configureSearch() {
+                    // Skip me
+                }
+            }, new IndexSettingsModule(index, settings),
 
-                    @Override
-                    protected void configure() {
-                        bindMapperExtension();
-                    }
-                }, new SearchModule(settings, namedWriteableRegistry) {
-                    @Override
-                    protected void configureSearch() {
-                        // Skip me
-                    }
-                }, new IndexSettingsModule(index, settings),
-
-                new AbstractModule() {
-                    @Override
-                    protected void configure() {
-                        bind(ClusterService.class).toProvider(Providers.of(clusterService));
-                        bind(CircuitBreakerService.class).to(NoneCircuitBreakerService.class);
-                        bind(NamedWriteableRegistry.class).toInstance(namedWriteableRegistry);
-                    }
-                }).createInjector();
+            new AbstractModule() {
+                @Override
+                protected void configure() {
+                    bind(ClusterService.class).toProvider(Providers.of(clusterService));
+                    bind(CircuitBreakerService.class).to(NoneCircuitBreakerService.class);
```

Comment:
can this be a `toInstance` and also can you try to remove the provider on clusterService as well?